ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot

```diff
@@ -97,12 +96,14 @@ Logout From RHODS Dashboard
 
 Wait For RHODS Dashboard To Load
     [Arguments]  ${dashboard_title}="${ODH_DASHBOARD_PROJECT_NAME}"    ${wait_for_cards}=${TRUE}
-    ...          ${expected_page}=Enabled    ${timeout}=60
+    ...          ${expected_page}=None    ${timeout}=60
```

Comment:
I'm not sure if `None `and `"${NONE}"` is the same thing.

Reply:
Yeah, just realized that we use the latter in other places, so I will fix; but it worked for my local testing :slightly_smiling_face: